src/lib_json/json_value.cpp

```diff
@@ -1396,7 +1397,7 @@ String Value::Comments::get(CommentPlacement slot) const {
 
 void Value::Comments::set(CommentPlacement slot, String comment) {
   if (!ptr_) {
-    ptr_ = std::make_unique<Array>();
+    ptr_ = std::unique_ptr<Array>(new Array());
```

Comment:
I think we should make a version of `make_unique` in this file with internal linkage.

It can forward to `std::make_unique` if `__cplusplus` indicates that we're on C++14. Then we won't have to deal with the `make_unique` suggestion again. People are going to keep running jsoncpp through linters and sending PRs if we keep the C++11 raw `new` `unique_ptr` creation idiom in place. :)